frontend/src/routes/Applications/AdvancedConfiguration.tsx

```diff
@@ -465,11 +465,13 @@ export default function AdvancedConfiguration(props: AdvancedConfigurationPagePr
             tooltip: t(
               'Displays the number of remote and local clusters where resources for the subscription are deployed. Click to search for all related clusters.'
             ),
-            exportContent: () => {
+            exportContent: (resource) => {
               const clusterCount = {
                 localPlacement: false,
                 remoteCount: 0,
               }
+              // mutates clusterCount object
+              getSubscriptionClusterCount(resource, clusterCount, true)
```

Comment:
`getSubscriptionClusterCount` is used in the above template to update the clusterCount object. In my first implementation, I had not realized the function had a mutative effect on it, so I've applied it here in the export callback, as to mirror it's affects, without using the link that it returns, as that is not necessary for the exporter.

Reply:
Thanks for including the explanatory comment.